applications/adv_networking_bench/README.md

```diff
@@ -1,48 +1,54 @@
-by batching size and whether GPUDirect is enabled.
+[here](/tutorials/high_performance_networking/README.md). With the system tuned, the application performance
+will be dictated by batching size and whether GPUDirect is enabled.
 
 At this time both the transmitter and receiver are written to handle an Ethernet+IP+UDP packet with a
 configurable payload. Other modes may be added in the future. Also, for simplicity, the transmitter and
-receiver must be configured to a single packet size.
+receiver are configured to a single packet size.
 
 ## Transmit
 
 The transmitter sends a UDP packet with an incrementing sequence of bytes after the UDP header. The batch
-size configured dictates how many packets the benchmark operator sends to the advanced network operator
-in each tick. Typically with the same number of CPU cores the transmitter will run faster than the receiver,
-so this parameter may be used to throttle the sender somewhat by making the batches very small.
+size configured dictates how many packets the benchmark operator sends to the NIC in each tick. Typically with
+the same number of CPU cores the transmitter will run faster than the receiver, so this parameter may be used
+to throttle the sender somewhat by making the batches very small.
 
 ## Receiver
 
-The receiver receives the UDP packets in either CPU-only mode or header-data split mode. CPU-only mode
-will receive the packets in CPU memory, copy the payload contents to a host-pinned staging buffer, and
-freed. In header-data split mode the user may configure a split point where the bytes before that point
-are sent to the CPU, and all bytes afterwards are sent to the GPU. Header-data split should achieve higher
-rates than CPU mode since the amount of data to the CPU can be orders of magnitude lower compared to running
-in CPU-only mode.
+The receiver receives the UDP packets in either CPU-only mode, header-data split mode, or GPU-only mode.
+- CPU-only mode will receive the packets in CPU memory, copy the payload contents to a host-pinned staging buffer, and
+  freed.
+- Header-data split mode: the user may configure separate memory regions for the header and data. The header is
+  sent to the CPU, and all bytes afterwards are sent to the GPU. Header-data split should achieve higher
+  rates than CPU mode since the amount of data to the CPU can be orders of magnitude lower compared to running
+  in CPU-only mode.
+- GPU-only mode: all bytes of the packets are received in GPU memory.
 
 ### Configuration
 
 The application is configured using a separate transmit and receive file. The transmit file is called
 `adv_networking_bench_tx.yaml` while the receive is named `adv_networking_bench_rx.yaml`. Configure the
-advanced networking operator on both transmit and receive per the instructions for that operator.
+Advanced Network library on both transmit and receive per the instructions for that library.
```

Comment:
This isn't quite right since there's only one library now, but in the past there were multiple operators. I think we should reword this so it's something like "for the YAML":